universal.go

```diff
@@ -49,7 +49,7 @@ type UniversalOptions struct {
 	MasterName string
 }
 
-func (o *UniversalOptions) cluster() *ClusterOptions {
+func (o *UniversalOptions) Cluster() *ClusterOptions {
```

Comment:
Please add something simple like `Cluster returns Redis Cluster options created from the universal options`. If you have better version - use it :)